packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text, Button, Modal } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { localize, Localize } from '@deriv/translations';
+import WalletsImage from 'Assets/svgs/wallets';
+import './wallets-upgrade-modal.scss';
+import { useWalletMigration } from '@deriv/api';
+
+const WalletsUpgradeModal = () => {
+    const { traders_hub, ui } = useStore();
+    const { toggleWalletsUpgrade } = traders_hub;
+    const { is_mobile, is_desktop } = ui;
+    const { is_eligible } = useWalletMigration();
```

Comment:
Wouldn't this hook make an extra `authorize` request?

Reply:
At the moment it might or might not cause extra authorisation call, depending when exactly its gonna be called (if its gonna be called shortly after another authorisation call, it wont trigger extra one, if its gonna be called in long enough period after initial authorisation, it will call it again). 

In ideal scenario, we have some sort of central state (e.g. context) which handles authorization and provides it to everything else without unecessary requests, but I would say, for time being, probably its unavoidable side effect :(

Reply:
it only call it once

![image](https://github.com/binary-com/deriv-app/assets/125247833/bf9867f3-d8d1-4f90-ab74-6f7e3271894b)

Reply:
Latest change to getActiveLoginIDFromLocalStorage should resolve this iinm 👍